File: src/projects/question/src/main/java/com/krm3d/question/domain/entities/Question.java
package com.krm3d.question.domain.entities;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;
import org.springframework.data.mongodb.core.mapping.Field;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Document
public class Question {
    @Id
    private String id;
    @Field
    private String questionTitleId;
    @Field
    private String questionName;
}

File: src/projects/ChildMaterializedView/Domain/Entities/Question.cs
using Core.Persistance.Repositories.EntityFramework;

namespace Domain.Entities
{
    public class Question : Entity
    {
        public string Id { get; set; }
        public string QuestionTitleId { get; set; }
        public string QuestionName { get; set; }
        public QuestionTitle QuestionTitle { get; set; }
        public ICollection<QuestionAnswer> QuestionAnswers { get; set; }
        public Question()
        {

        }

        public Question(string id, string questionTitleId, string questionName):this()
        {
            Id = id;
            QuestionTitleId = questionTitleId;
            QuestionName = questionName;
        }
    }
}
